Backends/RasterImage/output-to-image.lisp

```diff
@@ -12,7 +12,7 @@
             (,exit-fn (sheet stream)
               (declare (ignorable stream))
               (climi::write-bitmap-file
-               (image-mirror-image (sheet-mirror sheet))
+               (image-mirror-image sheet)
                ,stream :format ,format))
             (,enter-fn (sheet stream)
```

Comment:
GitHub doesn't let me comment at the actual location, but another new warning for this file is: `In DEFMETHOD CLIM-BACKEND:PORT-DEALLOCATE-PIXMAP (MCCLIM-RASTER-IMAGE::RGB-IMAGE-PORT: undefined function: MCCLIM-RASTER-IMAGE::PIXMAP-MIRROR`

Reply:
fixed (missing import).